src/pack/ekat_pack_macros.hpp

```diff
@@ -1,5 +1,5 @@
-#ifndef EKAT_MACROS_HPP
```

Comment:
The name was a bit generic. Since it only contained macros related to vectorization, I changed it to something more specific.